### ISSUE TITLE
Settlers die after loading a self created map (with original s3 editor)

### ISSUE THREAD

**Issue by LexoSettlers3** [jsettlers/settlers-remake#717]
Same map as in issue #716 but different bug:
I tried to load the map and this time no error occurs but directly after loading all my workers (delivery boys, builder and flattener) die. The soldiers, mine worker & co survive. But I´ve had the same bug a few days ago with all my soldiers dying.
OS: Win 8.1
JRE: 9.0.1
JSettlers - Version: 0c7a197

Hope this helps you!

Logfile: [2018-05-02_17-09-25_s3Fanmap2.0.zip](https://github.com/jsettlers/settlers-remake/files/1967852/2018-05-02_17-09-25_s3Fanmap2.0.zip)
The Map:  [the map.zip](https://github.com/jsettlers/settlers-remake/files/1967853/the.map.zip)

**Comment by kichooo** [comment]
I get the same error, with this map https://mapbase.siedler3.net/20er-russisches-roulette-2

**Comment by daUser** [comment]
I have the same issue, but with the build-in-Maps "Demo" and "400-2-Close". Maybe all other too, don't tested.

Version: c3add121
Logfile: [logs.zip](https://github.com/jsettlers/settlers-remake/files/2454075/logs.zip)

**Comment by kichooo** [comment]
Yep, I have the same version. Also reproduces on demo map. All I need to do is start, save game, load and it happens.

OS: Ubuntu 18.04.1 LTS

openjdk 10.0.2 2018-07-17
OpenJDK Runtime Environment (build 10.0.2+13-Ubuntu-1ubuntu0.18.04.2)
OpenJDK 64-Bit Server VM (build 10.0.2+13-Ubuntu-1ubuntu0.18.04.2, mixed mode)

**Comment by kichooo** [comment]
This bug is so hilarious I decided to [record](https://www.youtube.com/watch?v=ByglH2xt_xA&feature=youtu.be) it:

**Comment by paulwedeck** [comment]
I tried to reproduce your error(s), but both maps are working fine on the latest revision (master branch).

**Comment by kichooo** [comment]
Yep, I just built from latest master and it works :+1: 

Maybe it would be time to release new version. I think it can be closed.
